tests/next_tests/unit_tests/embedded_tests/test_nd_array_field.py

```diff
@@ -468,10 +468,10 @@ def test_absolute_indexing_value_return():
     field = common._field(np.reshape(np.arange(100, dtype=np.int32), (10, 10)), domain=domain)
 
     named_index = ((IDim, 12), (JDim, 6))
-    value = field[named_index]
+    value = field[named_index].ndarray
 
-    assert isinstance(value, np.int32)
-    assert value == 21
+    assert isinstance(value, np.ndarray)
+    assert value == np.asarray([21])
```

Comment:
If the expected value is a 0d field, shouldn't it be expressed as below?
```suggestion
    assert value == np.asarray(21)
```
By automatic shape broadcasting most likely works in both ways, but it feels it would be more correct to use a 0d array as reference, or am I misinterpreting the test?